src/components/Errors/Errors.tsx

```diff
@@ -0,0 +1,69 @@
+import { ErrorSpec } from '../../types/ErrorSpec';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
+type Props = {
+  error: ErrorSpec | null;
+};
+
+export const Errors: React.FC<Props> = ({ error }) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className="notification is-danger is-light has-text-weight-normal"
+    >
+      <button data-cy="HideErrorButton" type="button" className="delete" />
+      {/* show only one message at a time */}
+      {(() => {
```

Comment:
Calling such a function is hard to read and hard to maintain. It is better to make it as a separate helper that will return the desired tag and this tag is ready to be inserted into the code.
Also, each case part differs only in text, so there is no need to repeat the same tags every time, it is enough to calculate the required text and wrap it in the required tag